pkg/lib/cockpit-components-terminal.tsx

```diff
@@ -120,71 +213,88 @@
         this.getText = this.getText.bind(this);
         this.setTerminalTheme = this.setTerminalTheme.bind(this);
 
-        const term = new Term({
-            cols: props.cols || 80,
-            rows: props.rows || 25,
-            screenKeys: true,
-            cursorBlink: true,
-            fontSize: props.fontSize || 16,
-            fontFamily: 'Menlo, Monaco, Consolas, monospace',
-            screenReaderMode: true,
-            showPastingModal: false,
-        });
+        if (this.props.state) {
+            cockpit.assert(!this.props.channel);
+            this.terminal_state = this.props.state;
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21680-b8771fcf-20250814-123207-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)